virtual-organizations/LIGO.yaml

```diff
@@ -272,8 +272,8 @@ DataFederations:
         AllowedOrigins:
           - CIT_LIGO_ORIGIN_TEST_WRITE
         AllowedCaches: *ligo-allowed-caches
-        Writeback: https://origin-writetest.ligo.caltech.edu:1095
-        DirList: https://origin-writetest.ligo.caltech.edu:1095
+        Writeback: https://origin-writetest.ligo.caltech.edu:8443
+        DirList: https://origin-writetest.ligo.caltech.edu:8443
```

Comment:
Since the Topology config here is just for the caches, I don't think we actually need these anymore. I imagine that it was just the clients that needed this bit

Reply:
Looks like it's only used by pre-Pelican clients, and also the director for marking pre-Pelican origins as supporting listings and writeback. I'll remove it.